### PR TITLE
event- event image upload & display

### PR DESCRIPTION
**Pull request by nikhilmaurya03**

### CODE REVIEW

**Review by rudraabhilash — COMMENTED** [review]
Please paste snapshots of local testing. It is considered as good practice.

File: event/views.py [hunk 1]
```diff
@@ -16,7 +18,7 @@ def event_form(request):
 @csrf_exempt
 def event_form_submit(request):
     if request.method == 'POST':
-        form = Event_Form(request.POST)
+        form = Event_Form(request.POST, request.FILES)
```
[the remark this answers is not in the document]
Comment (nikhilmaurya03) [reply event/views.py:21]: to upload files, we use "request.FILES" in views.py to access the file that are uploaded in form.
Screenshot 2024-06-07 150230
Screenshot 2024-06-07 150253
Screenshot 2024-06-07 150314
Screenshot 2024-06-07 150329
Screenshot 2024-06-07 150348
Screenshot 2024-06-07 150409

File: event/urls.py [hunk 2]
```diff
@@ -6,6 +6,9 @@
     path('event_submit/', views.event_form_submit, name='event_submit'),
     path('eventsubmit_form/', views.submited_form, name='eventsubmit_form'),
     path('event_details/', views.event_details, name='event_details'),
+    path('all_event_details/<str:event_name>',views.all_event_details, name='all_event_details'),
```
Comment (rudraabhilash) [inline event/urls.py:9]: "specific" keyword is more appropriate rather than "all"

File: event/views.py [hunk 3]
```diff
@@ -31,10 +33,36 @@ def submited_form(request):
...
-    mydetails= Event_Table.objects.all().values()
-    template = loader.get_template('submit.html')
+    mydetails= Event_Table.objects.values_list('id','event_name','event_date').distinct()
+    template = loader.get_template('event_name_details.html')
     context = {
         'mydata':mydetails,
     }  
     return HttpResponse(template.render(context, request))  
 
+def all_event_details(request, event_name):
```
Comment (rudraabhilash) [inline event/views.py:43]: "specific" keyword is more appropriate than "all".

File: event/views.py [hunk 4]
```diff
@@ -31,10 +33,36 @@ def submited_form(request):
...
+    mydetails= Event_Table.objects.filter(event_name=event_name)
+    
+    template = loader.get_template('submit.html')
```
Comment (rudraabhilash) [inline event/views.py:46]: "specific_event.html" keyword is more appropriate than "submit".

File: event/views.py [hunk 5]
```diff
@@ -31,10 +33,36 @@ def submited_form(request):
...
+    context = {
+        'mydata':mydetails,
+    }  
+    return HttpResponse(template.render(context, request)) 
+
+def event_image(request, id):
+    i = Event_Table.objects.filter(id=id)
```
Comment (rudraabhilash) [inline event/views.py:53]: name 'i' is not my favourite.

File: event/models.py [hunk 6]
```diff
@@ -12,6 +12,6 @@ class Event_Table(models.Model):
   event_cost = models.IntegerField(null=True)
   event_type = models.CharField(max_length=255)
   event_winner_id = models.IntegerField(null=True)
-  event_image_location = models.CharField(max_length=255)
+  event_image_location = models.ImageField(upload_to='event/static')
```
Comment (rudraabhilash) [inline event/models.py:15]: Nice!

**Review by rudraabhilash — COMMENTED** [review]
Looks good overall. Please add "pillow" to requirements file.

**Comment by rudraabhilash** [comment]
Closing this as another pull request was raised.

**Pull request closed without merging**